bokehjs/src/coffee/models/glyphs/annular_wedge.coffee

```diff
@@ -107,6 +107,14 @@ class AnnularWedgeView extends Glyph.View
   draw_legend: (ctx, x0, x1, y0, y1) ->
     @_generic_area_legend(ctx, x0, x1, y0, y1)
 
+  _scxy: (i) ->
+    r = (@sinner_radius[i] + @souter_radius[i])/2
+    a = (@_start_angle[i]  + @_end_angle[i])   /2
+    return {x: @sx[i] + r*Math.cos(a), y: @sy[i] + r*Math.sin(a)}
+
+  scx: (i) -> @_scxy(i).x
+  scy: (i) -> @_scxy(i).y
```

Comment:
> or caching

NO, NO, NO. No, caching. Unless this is the only way to make things work and we gave it an actual thought.

I added `get_anchor_point()` method, which should be implemented accordingly. It currently uses `scx()` and `scy()`, but that should be changed later. I didn't want to do too much unrelated rewriting in this PR.